lib/tasks/suggested_resources.rake

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# These define tasks for managing our SuggestedResource records.
+namespace :suggested_resources do
+  # While we intend to use Dataclips for exporting these records when needed,
+  # we do need a way to import records from a CSV file.
+  desc 'Replace all Suggested Resources from CSV'
+  task :reload, [:addr] => :environment do |_task, args|
+    raise ArgumentError.new, 'URL is required' unless args.addr.present?
+
+    raise ArgumentError.new, 'Local files are not supported yet' unless URI(args.addr).scheme
+
+    Rails.logger.info('Reloading all Suggested Resource records from CSV')
+
+    url = URI.parse(args.addr)
+
+    raise ArgumentError.new, 'HTTP/HTTPS scheme is required' unless url.scheme.in?(%w[http https])
```

Comment:
Does it make sense to only accept HTTPS? (I'd hope that any CSV we're loading is coming through HTTPS and not HTTP, but the Internet is weird sometimes.)

Reply:
Oh, good call. I wasn't thinking about local. I don't see it as a huge issue, fwiw -- no need to make a change.